### DISCUSSION TITLE
Styling radio buttons

### DISCUSSION THREAD

**Discussion by trickydisco78** [chakra-ui/chakra-ui#2128] [General] [answered]
Is it possible to style radio buttons in this style?

![Screenshot 2020-09-30 at 14 04 42](https://user-images.githubusercontent.com/3627429/94688892-191e7880-0326-11eb-95b7-6e88e25f5f9e.png)

Or is it a case it can't be done without too much custom work?

**Comment by ljosberinn** [comment, marked as answer]
answered on discord https://discordapp.com/channels/660863154703695893/660863154703695896/761532541702111242

**Reply by Astatine-213-Tian** [reply]
I have the same question here! Would you mind posting your solution here?
